File: PatchUpdate/BsPatch/app/src/main/java/com/maotou/bspatch/BsPatch.java
package com.maotou.bspatch;

/**
 * Created by wuchundu on 18-4-11.
 */
public class BsPatch {

    static{
        System.loadLibrary("bspatch");
    }

    public static native int applyPatch(String oldApk, String newApk, String patch);
}

File: PatchUpdate/BsPatchKotlin/app/src/main/java/com/maotou/bspatch/BsPatch.kt
package com.maotou.bspatch

/**
 * Created by wuchundu on 18-4-11.
 */
object BsPatch {

    init {
        System.loadLibrary("bspatch")
    }

    external fun applyPatch(oldApk: String, newApk: String, patch: String): Int
}
